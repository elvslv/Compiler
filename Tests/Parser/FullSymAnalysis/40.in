var i, j: integer;
begin
	repeat
	begin
	end;
	begin
	end
	until i > j;
end.