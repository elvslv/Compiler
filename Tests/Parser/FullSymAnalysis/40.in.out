var I : INTEGER;
var J : INTEGER;

begin            
|                
|___repeat       
|   |            
|   |___begin    
|   |   |        
|   |   |___begin
|   |   |   |    
|   |   |   end  
|   |   |        
|   |   |___begin
|   |   |   |    
|   |   |   end  
|   |   |        
|   |   end      
|   |            
|   until        
|   |            
|   |___>        
|       |        
|       |___I    
|       |        
|       |___J    
|                
end              
